arviz/plots/kdeplot.py

```diff
@@ -80,7 +80,7 @@ def plot_kde(
 
     Examples
     --------
-    Plot default KDE
+    Plot default KDE 
```

Comment:
Lint is failing here due to trailing space

Reply:
Thank you for pointing this out.